buddy_mentorship/settings/local.py

```diff
@@ -128,7 +128,7 @@
 AUTH_USER_MODEL = "users.User"
 
 # Redirect to home URL after login (Default redirects to /accounts/profile/)
-LOGIN_REDIRECT_URL = "/"
+LOGIN_REDIRECT_URL = "/profile/"
```

Comment:
Not sure how I feel about this behavior - for example, think of case where user clicks "Requests" and gets sent to login. Shouldn't we try to send them back to requests? Not a blocking issue.

Reply:
That will still work, this is only when they hit login